app/src/main/java/com/hyperether/getgoing/db/DbHelper.java

```diff
@@ -14,9 +13,14 @@
  */
 public class DbHelper {
 
+    public interface Callback {
+        public void handlerDone();
+    }
+
     private static final String DATABASE_NAME = "getgoing_db";
     private static DbHelper instance;
     private Handler mHandler;
+    private Callback callback;
```

Comment:
@NikolaHadzicNs this is not good idea to have callback as class element: It will work for first callback, and second one will overwrite first one which will never be called in that case. You can remove this.